src/components/table/mixins/sort.js

```diff
@@ -19,43 +19,54 @@ export default {
       if (!this.isSortable || !this.sortBy) {
         return
       }
+      const hasFilesOrFolders = this.data.some(i => i.type === "folder" || i.type === "file")
```

Comment:
Can't really dig into the code right now but since this is used only in `OcTableFiles` which will _always_ contain of either *"folder"s* or *"file"s* I don't see a case where this statement won't be true.

Would `hasFilesAndFolders` with a logical `&&` instead of the `||` make more sense maybe?

Reply:
yeah you are right, I will check if its only used in `OcTableFiles` and if it is I'll remove this check.

Reply:
ok i looked it up and you are right, its only use is with OcTableFiles, so I removed this check.